code/game/machinery/cryopod.dm

```diff
@@ -587,6 +587,8 @@
 				if(Gh.client && Gh.client.holder) //just in case someone has a byond name with @ at the start, which I don't think is even possible but whatever
 					to_chat(Gh, "<span style='color: #800080;font-weight: bold;font-size:4;'>Warning: Your body has entered cryostorage.</span>")
 	log_admin("<span class='notice'>[key_name(E)] entered a stasis pod.</span>")
+	if(SSticker.mode.name == "nuclear emergency" || SSticker.mode.name == "wizard")
```

Comment:
Can you please make this a var on the base gamemode datum, and just set it to true for nukies and wizard
```suggestion
	if(SSticker.mode.tdm_gamemode)
```